src/components/admin/previewComponent/noticePreview/NoticePreview.tsx

```
@@ -0,0 +1,21 @@
+import React from 'react';
+import * as S from './NoticePreview.styled';
+import { useGetNotice } from '../../../../hooks/user/useGetNotice';
+import line from '../../../../assets/line.svg';
+
+const NoticePreview = () => {
+  const { noticeData } = useGetNotice({ keyword: '', page: 1 });
+
+  return (
+    <S.Container>
+      {noticeData?.notices.map((notice) => (
+        <S.Wrapper>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)